src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -159,11 +143,39 @@ private static void setupLogger(Application application) {
         application.addComponentMenuElement(Logger.class, "OFF logging", (ActionEvent e) -> logger.setLevel(Level.OFF));
     }
 
+    private static void setupFeatures(Application application) {
+        application.addComponentMenu(Feature.class, "Features");
+        application.addComponentMenuElement(Feature.class, "Load secret command", new SelectLoadSecretCommandOptionListener());
+        application.addComponentMenuElement(Feature.class, "Load triangle command", new SelectLoadTriangleCommandOptionListener());
+        application.addComponentMenuElement(Feature.class, "Rotate command", new SelectRotateCurrentCommandOptionListener(CommandsFeature.getDriverCommandManager()));
+        application.addComponentMenuElement(Feature.class, "Run command", new SelectRunCurrentCommandOptionListener(DriverFeature.getDriverManager()));
+        application.addComponentMenuElement(Feature.class, "Count command", new SelectCurrentCommandCounter(CommandsFeature.getDriverCommandManager()));
+        application.addComponentMenuElement(Feature.class, "Load Macro", new SelectLoadMacroOptionListener(MacroFeature.getDriver(), CommandsFeature.getDriverCommandManager()));
+        application.addComponentMenuElement(Feature.class, "Clear Macro", new SelectClearMacroOptionListener(MacroFeature.getDriver(), CommandsFeature.getDriverCommandManager()));
+    }
+
     private static void setupDriverMonitor(Application application) {
         application.addComponentMenu(MonitorDriverDecorator.class, "Driver Monitor", 5);
         application.addComponentMenuElement(MonitorDriverDecorator.class, "Print report", (ActionEvent e) -> UsageMonitorManager.printReport());
     }
 
+    private static void setupMouseCheckbox(Application application) {
+        JPanel panel = application.getFreePanel();
+
+        JCheckBox mouseCheckbox = new JCheckBox("Enable mouse");
+        mouseCheckbox.setToolTipText("Enable manual drawing with mouse.");
+        mouseCheckbox.setBounds(0, 0, 100, 30);
+        mouseCheckbox.setCursor(new Cursor(12));
+
+        panel.add(mouseCheckbox, new GridBagConstraints(0, 0, 1, 1, 1.0, 1.0,
+                GridBagConstraints.NORTHWEST, GridBagConstraints.NORTHWEST, new Insets(0, 0, 0, 0), 0, 0));
+
+        CheckboxAction enableMouseAction = new CheckboxAction("Enable mouse");
+        mouseCheckbox.setAction(enableMouseAction);
+        enableMouseAction.addObserver(new MouseControlObserver(DriverFeature.getDriverManager(), application.getFreePanel()));
+        enableMouseAction.addObserver(new MouseControlLoggerObserver());
+    }
+
```

Comment:
Given the amount of code, design could propably benifit from introducing MouseFeature and moving some code there.